web-client/src/presenter/actions/CaseConsolidation/shouldSetupConsolidatedCasesAction.test.js

```diff
@@ -28,26 +27,58 @@ describe('shouldSetupConsolidatedCasesAction', () => {
       },
       state: {
         form: {
-          eventCode: SINGLE_DOCKET_RECORD_ONLY_EVENT_CODES[0],
+          eventCode: 'M083',
         },
       },
     });
 
     expect(pathNoStub).toHaveBeenCalled();
   });
 
-  it('should return the yes path when the eventCode is NOT one of SINGLE_DOCKET_RECORD_ONLY_EVENT_CODES', async () => {
+  it('should return the no path when the eventCode is one of ENTERED_AND_SERVED_EVENT_CODES', async () => {
```

Comment:
❓ Similar to above comment, could we do this:
```suggestion
  it.each(ENTERED_AND_SERVED_EVENT_CODES)('should return the no path when the eventCode is one of ENTERED_AND_SERVED_EVENT_CODES', async () => {
```

Reply:
hmmm...maybe i'm not awake yet, but how then do we reference each ENTERED_AND_SERVED_EVENT_CODES to pass into the actual test?

Reply:
Ah, instead of `it.each` I think we'd instead have to use a `forEach` 🚀